node/pkg/node/testdata/test.yaml

```diff
@@ -0,0 +1,2 @@
+ethRPC: "ws://eth-config-file:8545"
```

Comment:
How about putting the eth and solana contracts in this configuration file as well?

A comment at the top of this denoting you can use flags, env vars, or this config would also be nice for those not fully familiar with wormhole.